### PR TITLE
Readme file update

### PR DESCRIPTION
**Pull request by josephignace**
Update readme information for the neurodegenerative repo, including project , actions, and output descriptions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,13 +88,105 @@ Further details on creating the study population can be found in the [`OpenSAFEL
 
 The contents of this repository MUST NOT be considered an accurate or valid representation of the study or its purpose. This repository may reflect an incomplete or incorrect analysis with no further ongoing work. The content has ONLY been made public to support the OpenSAFELY [open science and transparency principles](https://www.opensafely.org/about/#contributing-to-best-practice-around-open-science) and to support the sharing of re-usable code for other subsequent users. No clinical, policy or safety conclusions must be drawn from the contents of this repository.
```
Comment (rdenholm) [inline README.md:89]: Repeat of above

File: README.md [hunk 2]
```diff
@@ -66,13 +88,105 @@ Further details on creating the study population can be found in the [`OpenSAFEL
...
-
-# Licences
+## Output
+
+### consort_\*.csv
+
+| Variable           | Description                                                    |
+|--------------------|----------------------------------------------------------------|
+|     Description    | criterion applied to cohort                                    |
+|     N_midpoint6    | number of people in the cohort after criterion applied time    |
```
Comment (rdenholm) [inline README.md:98]: At the top of this section add a sentence explaining what _midpoint6 mean (i.e. disclosure control to Ns)

File: README.md [hunk 3]
```diff
@@ -33,9 +49,15 @@ Below is a description of each action in the [`project.yaml`](./project.yaml). A
...
+    -   Used to elaborate consort table, using study population size after each quality assurance, inclusion and exclusion criterion is applied.
 -   `table1_{cohort}`
     -   Runs [`table1.R`](./analysis/table1.R) which calculates descriptive statistics for pre- and post-exposure events for all outcomes and subgroups.
     -   Used for Table 1 in the manuscript
+-   `extendedtable1_{cohort}`    
+    -   Runs [`extendedtable1.R`](./analysis/extendedtable1.R) which calculates descriptive statistics for pre- and post-exposure events for all outcomes and subgroups.
+    -   Used for Extended Table 1 in the manuscript
 -   `table2_{cohort}`
     -   Runs [`table2.R`](./analysis/descriptives/table2.R) which calculates pre- and post-exposure event counts and person days of follow-up for all outcomes and subgroups.
     -   Used for Table 2 in the manuscript
```
Comment (rdenholm) [inline README.md:63]: If you're going to add action descriptions, could we add make model input, reusable action location, and make model output

**Review by rdenholm — CHANGES_REQUESTED** [review]
Hi Jose, just a couple of notes to be complete.

File: README.md (hunk 3 above)
Comment (josephignace) [reply README.md:63]: The description for such actions is already there (see lines 64-74).

File: README.md (hunk 2 above)
Comment (josephignace) [reply README.md:98]: Added explanantion and links with information.

**Review by rdenholm — APPROVED** [review]

**Pull request merged**